src/GitVersionHelper.php

```diff
@@ -63,4 +63,36 @@ public static function getNameAndVersion()
     {
         return self::appName() . '/' . self::getVersion();
     }
+    
+    /**
+     * Returns comment hash with custom length
+     *
+     * @param null $length
+     * @return string
+     *
+     */
+    public static function getCommitHash($length=null)
```

Comment:
Sorry to change my mind again, but I think `getHash` would fit better alongside `getVersion`.